index.html

```diff
@@ -41,10 +41,20 @@ <h2>Why you should use it</h2>
 			</div>
 		</div>
 		<div class="row">
+			<div class="col-sm-4 feature">
+				<h3>Modern Features</h3>
```

Comment:
This is a nit, but it's been pestering me forever on the website: mind un-titlecasing these headings? Meaning "Modern features" instead of "Modern Features", etc.